modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbar.ts

```diff
@@ -47,11 +47,15 @@ const register = (editor: Editor, registryContextToolbars: Record<string, Contex
   const lastElement = Singleton.value<SugarElement<Element>>();
   const lastTrigger = Singleton.value<TriggerCause>();
   const lastContextPosition = Singleton.value<InlineContent.ContextPosition>();
+  const lastToolbarKey = Singleton.value<string>();
 
   const contextbar = GuiFactory.build(
     renderContextToolbar({
       sink,
       onEscape: () => {
+        if (lastToolbarKey.get().getOr('default') === 'quicklink') {
```

Comment:
It's kind of strange to default a value just to fail a predicate check. I would have expected that we'd have something on Optional that takes a value, and only returns true if it is some, and the some is that value, however, I can't find it.

Normally, this function is called `is(v: V)` and returns false for None, and true for Some if the value contained is v. But maybe it only exists in languages with an easier way of stating `Eq` instances for equality. 

I guess because we clearly don't have this function, it's probably *marginally* better to use `.exists((v) => v === 'quicklink')`, though I'm not sure if I can justify that.

Reply:
That was moved to `Optionals`
https://github.com/tinymce/tinymce/blob/ba6994f39b09ecd2c28cef68c3e8a85a4815e235/modules/katamari/src/main/ts/ephox/katamari/api/Optionals.ts#L8-L9